server/src/entity/Service.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  Entity,
+  JoinTable,
+  ManyToMany,
+  ManyToOne,
+  OneToMany,
+  PrimaryGeneratedColumn,
+  PrimaryColumn,
+  OneToOne,
+} from 'typeorm';
+import { Account } from './Account';
+import { Table } from './Table';
+
+export enum serviceType {
+  REFILL_DRINK = 'refill_drink',
+  SERVE_FOOD = 'serve_food',
+  TAKE_ORDER = 'take_order',
+  CHECK_ON_TABLE = 'check_on_table',
+  ALLERGEN_ASSISTANCE = 'allergen_assistance',
+  UNKNOWN = 'unknown',
+}
+
+@Entity()
+export class Service extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  public id: number;
+
+  @OneToOne((type) => Account, (account) => account.id)
+  public userId: string;
```

Comment:
Prefer using the type of the entity in the relationship and change the name to `user`. See http://typeorm.io/#/one-to-one-relations.

Reply:
This also isn't a proper way to specify the relationship according to typeorm. Look moreso at the documentation

Reply:
Tis now a one to many relationship.